plugins/MauticCrmBundle/Integration/SugarcrmIntegration.php

```diff
@@ -1449,7 +1449,9 @@ private function fetchDncToMautic(Lead $lead, array $data)
      * @param $leadSugarFields
      * @param string $object
      *
-     * @return mixed
+     * @return array The first element is made up of records that exist in Mautic, but which no longer have a match in CRM.
+     *               We therefore assume that they've been deleted in CRM and will mark them as deleted in the pushLeads function (~line 1320).
+     *               The second element contains Ids of records that were explicitly marked as deleted in CRM. ATM, nothing is done with this data.             
```

Comment:
```suggestion
     *               The second element contains Ids of records that were explicitly marked as deleted in CRM. ATM, nothing is done with this data.
```
There are also some spaces at the end of this line.